sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,10 +1,6 @@
 # Release History
 
-## 5.12.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 5.12.0 (2024-07-17)
```

Comment:
This should be 5.11.4, I believe.   We didn't add new functionality, just tuned some behavior and fixed up some bugs.

Reply:
I had that at first but doesn't it need to be higher than the beta version? I was getting a warning from the release script.

Reply:
Not according to Krzysztof.  He recommended using a 5.12.x beta version so that we could continue to build on the 11 line.   That said, if the tooling is complaining, then let's just go with 5.12.0 and not waste time fighting it.   It's a minor convention and IMHO, it isn't really an important distinction.